conf/prod.routes

```diff
@@ -0,0 +1,5 @@
+# Add all the application routes to the app.routes file
+->         /v2                         app.Routes
```

Comment:
this is ok for now for govuk notifications but when we add a stub for the external verification provider then we may have to rethink this